sbor/src/instruction_extractor.rs

```diff
@@ -1,5 +1,7 @@
-// Process events received from decoder and extract data related to instructions
-
+/// Process events received from SBOR decoder and extract data related to each instruction
```

Comment:
incorrect doc, if this doc is for this file, then you should use `//!`, or put docs on top of types.